ingestion_tools/scripts/common/config.py

```diff
@@ -248,8 +228,10 @@ def resolve_output_path(self, key: str, obj: BaseImporter) -> str:
             "tiltseries": "{dataset_name}/{run_name}/TiltSeries",
             "gain": "{dataset_name}/{run_name}/Frames/{run_name}_gain",
```

Comment:
> ### Gains
> Update gain files to be stored in /<dataset-id>/<run-name>/Gain
> Update the naming of the gain file to be <name-in-source>.<ext>

Reply:
Addressed in https://github.com/chanzuckerberg/cryoet-data-portal-backend/pull/271